# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux/hotfix-kern-cont-ia64.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- linux-6.8/arch/ia64/kernel/time.c.vanilla	2024-04-18 13:44:17.633984660 +0200
+++ linux-6.8/arch/ia64/kernel/time.c	2024-04-18 13:47:05.103983925 +0200
@@ -307,10 +307,10 @@
 
 	if (platform_base_drift != -1) {
 		itc_drift = platform_base_drift*itc_ratio.num/itc_ratio.den;
-		printk("+/-%ldppm\n", itc_drift);
+		printk(KERN_CONT "+/-%ldppm\n", itc_drift);
 	} else {
 		itc_drift = -1;
-		printk("\n");
+		printk(KERN_CONT "\n");
 	}
 
 	local_cpu_data->proc_freq = (platform_base_freq*proc_ratio.num)/proc_ratio.den;
